src/components/dashboard/dashboardItem/LoginTable.js

```diff
@@ -0,0 +1,84 @@
+/**
+ *
+ * @author mian
+ *
+ * A table of recent logins for the user.
+ *
+ */
+
+import React from "react";
+import { useQuery } from "react-query";
+
+import { logins, LOGINS_QUERY_KEY } from "serviceFacades/users";
+import ErrorTypographyWithDialog from "components/error/ErrorTypographyWithDialog";
+import { useTranslation } from "i18n";
+import {
+    formatDate,
+    getFormattedDistance,
+} from "components/utils/DateFormatter";
+import {
+    useTheme,
+    Skeleton,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow,
+    Paper,
+} from "@mui/material";
+
+export default function LoginsTable() {
+    const { t } = useTranslation("dashboard");
+    const { t: i18nCommon } = useTranslation("common");
+
+    const theme = useTheme();
+    const { status, data, error } = useQuery({
+        queryKey: [LOGINS_QUERY_KEY],
+        queryFn: () => logins({ limit: 5 }),
+    });
+
+    if (status === "error") {
+        return (
+            <div style={{ padding: theme.spacing(1) }}>
+                <ErrorTypographyWithDialog
+                    errorObject={error}
+                    errorMessage={t("loginsTableError")}
+                />
+            </div>
+        );
+    }
+    if (status === "loading") {
+        return <Skeleton variant="rectangular" height={200} />;
+    }
+    return (
+        <TableContainer component={Paper}>
+            <Table size="small">
+                <TableHead>
+                    <TableRow>
+                        <TableCell>{t("loginTime")}</TableCell>
+                        <TableCell></TableCell>
+                        <TableCell>{t("ipAddress")}</TableCell>
+                    </TableRow>
+                </TableHead>
+                <TableBody>
+                    {data["logins"].map((row) => (
+                        <TableRow>
```

Comment:
The browser dev console should be reporting a warning that these array elements each require a unique key. Since these rows aren't going to be dynamically rearranging, I think an `index` key should be fine here (unless the `row` has a `uuid`, which would be even better):
```suggestion
                    {data["logins"].map((row, index) => (
                        <TableRow key={index}>
```

Reply:
Ah right, of course. We do have a UUID but currently aren't passing it up from the backend. I'll probably just use an index as you say; it's true these shouldn't change after initial rendering.